### ISSUE TITLE
Test import bear plates 

### ISSUE THREAD

**Issue by thokall** [CGI-NRM/SeqDB-documentation#2]
In order to identify issues or any problem with the CGI importer 10 plates that have genotype data should be imported.

**Comment by Inkimar** [comment]
@thokall can this be closed ?

**Comment by thokall** [comment]
We have no formally done the 10 plates just yet, We now have all the data so unless we have another issue, lets close this issue once that is done.

**Comment by thokall** [comment]
Correction, We still have incomplete id information, but as I remember those could be altered in bulk afterwards without any major workarounds.

**Comment by thokall** [comment]
Finally first plate imported. I will open a new issue where we can discussi the possibility to update genotype data from bulk export/import.
